guides/_subsections/download-edit-and-upload-a-customized-event-template.md

```diff
@@ -142,3 +142,30 @@ delete **Custom Event Templates**.
     %}
   </li>
 </ol>
+
+### [Preconfiguring Event Templates within Cryostat](#preconfiguring-automated-rules-within-cryostat)
+
+If deploying **Cryostat** in a **Kubernetes** environment through the **Cryostat Operator**, custom event templates can be preconfigured within **Cryostat**.
+
+To begin, create a **ConfigMap** within **Kubernetes** or **Red Hat Openshift** from this text file
```

Comment:
"this text file" or `custom-template.xml` isn't directly referencing anything in the above guide content, which is pretty vague about the file being downloaded. Maybe updating the contents above to tell the user to not just download an example template but to rename it to `custom-template.xml` would help link these two parts together?